dev-python/pythondialog/pythondialog-3.5.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="A Python module for making simple text/console-mode user interfaces"
+HOMEPAGE="http://pythondialog.sourceforge.net/"
+SRC_URI="mirror://sourceforge/pythondialog//${PV}/python3-${P}.tar.bz2"
+
+LICENSE="LGPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~ia64 ~ppc ~sparc ~x86"
+IUSE="doc examples"
+REQUIRED_USE="examples? ( doc )"
+
+RDEPEND="dev-util/dialog"
+
+python_prepare_all() {
+	sed -e "/^    'sphinx.ext.intersphinx',/d" -i doc/conf.py || die
+	distutils-r1_python_prepare_all
+}
+
+distutils_enable_sphinx doc
+
+python_install_all() {
+	if use examples; then
```

Comment:
Install with `docs` IUSE to get rid of that weird `REQUIRED_USE`.

Reply:
just to clarify, the examples are installed by default, as EAPI6 has removed handling for examples from distutils-r1.eclass.  The docs are handled by `distutils_enable_sphinx`, but that doesn't work for the examples dir.  See the latest iteration for example.